app/views/shared/_header.html.erb

```diff
@@ -2,6 +2,6 @@
   <%= current_user.email %>
   <%= link_to "Recipes", recipes_path, class: 'btn' %>
   <%= link_to "Food", foods_index_path, class: 'btn' %>
-  <button>Inventory</button>
+  <%= link_to "Inventory", inventories_path, class: 'btn' %>
   <%= button_to "Log out", destroy_user_session_path, method: :delete, data: { turbo: false }, class: 'btn'  %>
```

Comment:
@taldr27 I faced issues changing the button from a "button_to" to a "link_to". Would you please point me in the right direction on how to fix this issue? I don't know how to fix it.

Reply:
Sure, you can use `<%= link_to "Log out", destroy_user_session_path, method: :delete %>` instead of the `button_to` tag.
This will create a link that, when clicked, will send a `DELETE` request to the destroy action in the sessions controller, which is the default `Devise` controller for handling user authentication. You can read more about `link_to` tag [here](https://www.rubyguides.com/2019/05/rails-link_to-method/).